docs/managing_deis/upgrading-deis.rst

```diff
@@ -63,6 +63,86 @@ Finally, update ``deisctl`` to the new version and reinstall:
     enable PROXY protocol for ports 80 and 443 on the ELB, add a ``TCP 443:443`` listener, and
     change existing targets and health checks from HTTP to TCP.
 
+
+Graceful Upgrade
+----------------
+
+Alternatively, an experimental feature exists to provide the ability to perform
+a graceful upgrade. This process is available for those upgrading from version
+1.9.0 or greater and is intended to facilitate upgrades within a major version.
+Upgrading between major versions is not supported. Unlike the in-place upgrade
+process described above, this process keeps the platform's routers and
+publishers up during most of the upgrade process to minimze application
+downtime.
+
+.. note::
+
+    The process utilizes a rolling restart of the platform's router components.
+    No more than one router instance is ever down at a time, but each stops and
+    restarts in fairly rapid succession.  Therefore, your load balancer
+    configuration is the determining factor in how much downtime, if any, will
+    occur during a successful upgrade.
+
+    Load balancers that are quick to remove a host from service after just one
```

Comment:
A bit of a run-on. I think the original wording might have been easier to on the eyes.